docs/specification/objects.md

```diff
@@ -317,35 +317,37 @@ len(`name`) | <=    | 128, UTF-8 Encoded
 ---
 ## Ingredient Run
 
-A particular instance of an ingredient spec.
-
-Field name | Value type | Default | Description
------------|------------|---------|------------
-`uids`         | Map[String, String] | Empty | A collection of [Unique Identifiers](../unique-identifiers)
-`type`         | String     | Req. | "ingredient_run"
-`material`     | [Material Run](./#material-run) | Req. | Material that is this ingredient
-`process`      | [Process Run](./#process-run) | Req. | Process that the ingredient is used in
-`notes`       | String     | None | Some free-form notes about the run.
-`tags`        | Set[String]| Empty | [Tags](../tags)
-`file_links`  | Set\[[File Links](../file-links)] | Empty | Links to associated files, with resource paths into the files API
-`mass_fraction` | [Real Value](../value-types#real-values) | None | The mass fraction of the ingredient in the process
-`volume_fraction` | [Real Value](../value-types#real-values) | None | The volume fraction of the ingredient in the process
-`number_fraction` | [Real Value](../value-types#real-values) | None | The number fraction of the ingredient in the process
-`absolute_quantity` | [Real Value](../value-types#real-values) | None | The absolute quantity of the ingredient in the process
-`spec`| [Ingredient Spec](./#ingredient-spec) | Req. | The spec of which this is a run
+A particular instance of an ingredient spec. Note that the `name` and `labels` for an Ingredient Spec are shared with all associated Ingredient Runs.
+
+*Due to complexities of implementation, care must be taken in constructing Ingredient Runs.*  See [Known Limitations](../../known_limitations).  
```

Comment:
`care must be taken` seems unnecessarily opaque.  You could reference the change to this spec when name/labels were removed and mention that some implementations may still have name/labels on `IngredientRun`.